src/api/middleware/utils.js

```diff
@@ -81,9 +82,19 @@ const when = (
     return next();
 };
 
+/**
+ * Checks if a god token was sent in the request, indicating that the user might be in god mode.
+ */
+const godMode = (req, _, next) => {
+    if (req.body?.god_token === config.god_token) req.godMode = true;
+
+    return next();
+};
+
```

Comment:
So, if I understood correctly, this logic should be inside `src/middleware/auth.js`, right? 
Also, should I change every `isAdmin, isGod` that might be inside a call to the `or` util to `hasAdminPermissions`, since those are already going to be covered by it ?

Reply:
Yes, sorry I had the wrong link for the code.

No don't change the validation, these are different things. In this case you won't validate anything, you'll simply provide a utility variable for future code to depend on it. It can be useful for scenarios where you can be a company account with permissions vs. an admin/god and we need to change behavior based on which of these the user is.

Besides, changing the validators to use the hasAdminPermissions defeats the purpose of the creation of the `or` util, which was built to give more context to the reader about what validation is doing, and modularity.